### DISCUSSION TITLE
does floccus support syncing Firefox keywords across computers?

### DISCUSSION THREAD

**Discussion by cimenta** [floccusaddon/floccus#1365] [Q&A] [answered]
hi
I am using floccus on my computer with 4 different browsers - Firefox with keywords, Chrome, Vivaldi, Brave. Keyword are working fine.

Now I installed Floccus on new computer and the keyords are lost. Not sure if it matters but I installed Floccus on Chrome fist on the new computer. Not sure about that.

Thank you
Radek

**Comment by marcelklehr** [comment, marked as answer]
Floccus is not able to sync keywords in any direction on any browser at the moment.
